advanced/events.md

```diff
@@ -252,12 +252,12 @@ tinymce.init({
 
 ### Redo
 
-This event gets fired when redo is made by the user.
+This event gets fired when a request to redo the changes made by the Undo event is made by the user.
```

Comment:
The previous version is simpler

Reply:
This event gets fired when a request to redo is made by the user.

Reply:
Changed as suggested by @techtangents .